[PPC64] "No supported platform for SIMD vectorization!"

Hi all.

Does `seqan2` not support PowerPC64 architecture?

```
[ 28%] Building CXX object apps/razers3/CMakeFiles/razers3.dir/razers.cpp.o
cd /builddir/build/BUILD/seqan-8a875d71478a464de5ac3bcf2707f46819285a1f/build/Release/apps/razers3 && /usr/bin/c++  -DSEQAN_APP_VERSION=\"3.5.7\" -DSEQAN_DATE=\"\" -DSEQAN_DISABLE_VERSION_CHECK -DSEQAN_HAS_BZIP2=1 -DSEQAN_HAS_EXECINFO=1 -DSEQAN_HAS_OPENMP=1 -DSEQAN_HAS_ZLIB=1 -DSEQAN_REVISION=\"tarball\" -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -I/builddir/build/BUILD/seqan-8a875d71478a464de5ac3bcf2707f46819285a1f/include  -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -m64 -fPIC -Wl,-z,now  -W -Wall -pedantic -fopenmp -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -m64 -fPIC -Wl,-z,now -DNDEBUG    -DSEQAN_GLOBAL_EXCEPTION_HANDLER=1 -o CMakeFiles/razers3.dir/razers.cpp.o -c /builddir/build/BUILD/seqan-8a875d71478a464de5ac3bcf2707f46819285a1f/apps/razers3/razers.cpp
make[2]: Leaving directory '/builddir/build/BUILD/seqan-8a875d71478a464de5ac3bcf2707f46819285a1f/build/Release'
In file included from /builddir/build/BUILD/seqan-8a875d71478a464de5ac3bcf2707f46819285a1f/include/seqan/simd.h:118:0,
                 from /builddir/build/BUILD/seqan-8a875d71478a464de5ac3bcf2707f46819285a1f/include/seqan/align.h:56,
                 from /builddir/build/BUILD/seqan-8a875d71478a464de5ac3bcf2707f46819285a1f/include/seqan/store.h:38,
                 from /builddir/build/BUILD/seqan-8a875d71478a464de5ac3bcf2707f46819285a1f/apps/razers3/razers.cpp:63:
/builddir/build/BUILD/seqan-8a875d71478a464de5ac3bcf2707f46819285a1f/include/seqan/simd/simd_base_seqan_impl.h:52:4: warning: #warning is a GCC extension
   #warning "No supported platform for SIMD vectorization!"
    ^~~~~~~
/builddir/build/BUILD/seqan-8a875d71478a464de5ac3bcf2707f46819285a1f/include/seqan/simd/simd_base_seqan_impl.h:52:4: warning: #warning "No supported platform for SIMD vectorization!" [-Wcpp]
{standard input}: Assembler messages:
{standard input}:1037212: Warning: end of file not at end of a line; newline inserted
{standard input}:1037213: Error: bad register expression
{standard input}:1037213: Error: missing separator
{standard input}: Error: open CFI at the end of file; missing .cfi_endproc directive
c++: internal compiler error: Killed (program cc1plus)
Please submit a full bug report,
with preprocessed source if appropriate.
See <http://bugzilla.redhat.com/bugzilla> for instructions.
make[2]: *** [apps/razers3/CMakeFiles/razers3.dir/build.make:66: apps/razers3/CMakeFiles/razers3.dir/razers.cpp.o] Error 4
make[1]: *** [CMakeFiles/Makefile2:2620: apps/razers3/CMakeFiles/razers3.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
```

Full build log: https://kojipkgs.fedoraproject.org//work/tasks/3197/24213197/build.log

--- a/include/seqan/simd/simd_base_seqan_impl.h
+++ b/include/seqan/simd/simd_base_seqan_impl.h
@@ -48,8 +48,12 @@
 #elif defined(PLATFORM_GCC) && (defined(__x86_64__) || defined(__i386__))
   /* GCC-compatible compiler, targeting x86/x86-64 */
   #include <x86intrin.h>
-#else
-  #warning "No supported platform for SIMD vectorization!"
+#elif defined(SEQAN_SIMD_ENABLED)
+  #pragma message "You are trying to build with -DSEQAN_SIMD_ENABLED, which might be " \
+  "auto-defined if AVX or SSE was enabled (e.g. -march=native, -msse4, ...), " \
+  "but we only support x86/x86-64 architectures for SIMD vectorization! " \
+  "You might want to use UME::SIMD (https://github.com/edanor/umesimd) combined " \
+  "with -DSEQAN_UMESIMD_ENABLED for a different SIMD backend."
 #endif
 
 namespace seqan {

